src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -227,6 +227,13 @@ string MakeHelixWorkItemProject(WorkItemInfo workItemInfo)
                     command.AppendLine($"{setEnvironmentVariable} DOTNET_EnableCrashReport=1");
                 }
 
+                // https://github.com/dotnet/runtime/issues/97186#issuecomment-1899748532
+                // Helping to see if this is reproducible on Linux by disabling GC regionns
+                if (isUnix)
+                {
+                    command.AppendLine($"{setEnvironmentVariable} DOTNET_GCName=clrgc");
```

Comment:
I think this should be `libclrgc.so`? Not sure it will resolve as-is. cc @cshung

Reply:
If it doesn't resolve will the runtime error or fall back?

Reply:
Looks like it fails to start

```
/datadisks/disk1/work/9F910860/p/dotnet-cli/sdk/8.0.100/vstest.console.dll
+ dotnet exec /datadisks/disk1/work/9F910860/p/dotnet-cli/sdk/8.0.100/vstest.console.dll @vstest_1.rsp
GC initialization failed with error 0x8007007E
Failed to create CoreCLR, HRESULT: 0x8007007E
```

Reply:
Trying again with `libclrgc.so`

Reply:
Looks like Linux passes with that setting enabled.